ivy_tests/test_ivy/test_frontends/test_numpy/test_mathematical_functions/test_rounding.py

```diff
@@ -58,3 +58,35 @@ def test_numpy_ceil(
         subok=True,
         test_values=False,
     )
+
+
```

Comment:
can you add the `@handle_cmd_line_args` here?

Reply:
Done